crates/smartmodule-development-kit/src/main.rs

```diff
@@ -10,6 +10,7 @@ mod set_public;
 use std::path::{PathBuf};
 use clap::Parser;
```

Comment:
Minor nit. can you separate std lib from third party?

Reply:
I have just updated it. Thanks